composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "symfony/dependency-injection": "~2.0|~3.0"
+        "php":                          "^5.3.9|^7.0",
+        "symfony/dependency-injection": "^2.7|^3.0|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0",
-        "symfony/config": "~2.0|~3.0",
-        "symfony/expression-language": "~2.4|~3.0"
+        "phpunit/phpunit":              "^4.8|^5.7",
```

Comment:
Since this is a dev requirement we don't need to support multiple phpunit versions, right?